source/com.microsoft.tfs.client.common/src/com/microsoft/tfs/client/common/credentials/CredentialsHelper.java

```diff
@@ -114,11 +119,15 @@ public static Credentials getOAuthCredentials(final URI serverURI, final Action<
                 authenticator.getUriToKeyConversion().convert(serverURI, authenticator.getAuthType());
             removeStalePersonalAccessToken(tokenKey, serverURI);
 
+            final TokenPair oauth2Token =
+                (accessToken == null) ? null : new TokenPair(accessToken.getAccessToken(), "null"); //$NON-NLS-1$
```

Comment:
This is because the constructor for `TokenPair` doesn't take empty or null strings.  It must have a value.